File: app/src/main/java/com/vpulse/ftpnext/database/TableTest1/TableTest1.java
package com.vpulse.ftpnext.database.TableTest1;

import com.vpulse.ftpnext.database.ABaseTable;

import org.jetbrains.annotations.NotNull;

public class TableTest1 extends ABaseTable {

    private int mValue = 0;

    public TableTest1() {
    }

    public TableTest1(int iValue) {
        mValue = iValue;
    }

    public void setValue(int iValue) {
        mValue = iValue;
    }

    public int getValue() {
        return mValue;
    }

    @Override
    protected void setDataBaseId(int iDataBaseId) {
        mDataBaseId = iDataBaseId;
    }

    @NotNull
    @Override
    public String toString() {
        return "test";
    }
}

File: app/src/main/java/com/vpulse/ftpnext/database/TableTest1/TableTest1.kt
package com.vpulse.ftpnext.database.TableTest1

import com.vpulse.ftpnext.database.ABaseTable

class TableTest1 : ABaseTable {
    var value: Int = 0

    constructor() {}
    constructor(iValue: Int) {
        value = iValue
    }

    public override fun toString(): String {
        return "test"
    }
}